Specs/Scene/Model/ModelDrawCommandSpec.js

```diff
@@ -26,775 +26,300 @@ import {
   WebGLConstants,
 } from "../../../Source/Cesium.js";
 
-describe("Scene/Model/ModelDrawCommand", function () {
```

Comment:
Just as a note: all I did was add "WebGL" at the bottom of the file, which made the indents different.